code/game/objects/items/melee/energy.dm

```diff
@@ -274,4 +274,4 @@
 	name = "hardlight blade"
 	desc = "An extremely sharp blade made out of hard light. Packs quite a punch."
 	icon_state = "lightblade"
-	item_state = "lightblade"
+	item_state = "lightblade"
```

Comment:
newline

Reply:
Can you explain to me what you mean by that and how to fix that?

Reply:
I think they mean add a newline at the end of the file. It's not something you have to worry about though